web/src/context/NewDisputeContext.tsx

```diff
@@ -62,14 +65,13 @@ const initialDisputeData: IDisputeData = {
   title: "",
```

Comment:
I have been preferring the following solution over creating an object for context initialization:
```typescript
const NewDisputeContext = createContext<INewDisputeContext | undefined>(undefined)

export const useNewDisputeContext = () => {
  const context = useContext(NewDisputeContext);
  if (!context) {
    throw new Error("Context Provider not found.");
  }
  return context;
};
```
wdyt?

Reply:
hmm, i agree with the check. 
Also reduces the code size 👍